src/08-accessing.js

```diff
@@ -32,7 +32,10 @@
  *  getFirstName(person);
  *  //> "Rachel"
  */
-function getFirstName(person) {}
+function getFirstName(person) {
+  
```

Comment:
Let's lose this empty line as it doesn't separate any code worth breaking apart.